### ISSUE TITLE
Function call stays in one line that exceeds printWidth

### ISSUE THREAD

**Issue by SutnuiBorshik** [prettier/prettier#12892]
**Prettier 2.6.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBnOMAqBLAtnCAVxgAoSBKAAgF4A+S4AHSkssilQgBs4A6LiAHMSAcnioYI8gG5mAXwA0lTvgBihKGBjZoAYQCGXLiWasAbnABOATwAy0QQAV9l-fniWAcm7gBGBaaUFjb2UE4uPh7e+ABMASxBVnYOzq7uVtFwAMzx5kmh4WkYGT4ALPHkMiAKIBAADtrQqMigLpYQAO6pCM0ohh361s01AEauYADWGADKPrbYUHDIAGaG6KPjUzDTdfpgC4LIMJaEcDVwuCNwACbXN7b6YYT6gnCqEJa4+jDaYcgg+mIEGqIAAFjBcFwAOqg7DiXZgODTHpw7BmOHWf5gVDDEALdCWGCOVyCL4rNZnEAAK1QAA9pgceABFQgQeDkrjrEC7SwE-4wax1OCoMCWbANEF1MWwKHYa4wUHIAAcAAYalKIOgoa46v8pcKrBYQQBHVnwYn1XoA1AAWkWNxuIMscFN2GdxJeZKQq05lPQuGwRxOfsZcBZbKW3opNRg+hGsvliqQcRAx302C4B10EFwXpAwoArCDCOhMHHej6uWZTgBJKB3WDTUXimAAQXr0wFPA56DkciAA)

```sh
--parser typescript
```

**Input:**

```tsx
setTimeout(() => {
  console.log('test');
}, someFunctionCall(
  veryLongParameterName1,
  veryLongParameterName2,
  veryLongParameterName3,
  veryLongParameterName4,
));
```

**Output:**

```tsx
setTimeout(() => {
  console.log("test");
}, someFunctionCall(veryLongParameterName1, veryLongParameterName2, veryLongParameterName3, veryLongParameterName4));

```
`someFunctionCall` becomes one line no matter what. You can pass other functions or expressions as it's params and it doesn't matter.

**Expected behavior:**
Function call should be broken into multiple lines. `printWidth` should be enforced.

**Comment by hansireit** [comment]
If no one is working on this I will take a look at this.

**Comment by thorn0** [comment]
fixed by #13341
